src/main/java/io/dashboardhub/pipelinedashboard/config/UsernameAuditorAware.java

```diff
@@ -0,0 +1,19 @@
+package io.dashboardhub.pipelinedashboard.config;
+
+import org.springframework.data.domain.AuditorAware;
+import org.springframework.security.core.Authentication;
+import org.springframework.security.core.context.SecurityContextHolder;
+
+public class UsernameAuditorAware implements AuditorAware<String> {
```

Comment:
I am not sure about the naming of the end of the file. Would it not be better placed in an `Auditor` directory?